fee_allocator/fees_collected/fees_2024-07-04_2024-07-18.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mainnet": 251328.80,
+  "arbitrum": 36140.14,
+  "polygon": 12540.96,
```

Comment:
Raw amount for Poly is 15528223994 so should be 15528.22?

Reply:
Note the comment at the top. I don't think this matters anymore after we refactored the aura redistribution.